corehq/apps/domain/utils.py

```diff
@@ -100,3 +101,20 @@ def get_domain_url_slug(hr_name, max_length=25, separator='-'):
             text += separator + word
 
     return text[:max_length]
+
+
+def guess_domain_language(domain_name):
+    """
+    A domain does not have a default language, but its apps do. Return
+    the language code of the most common default language across apps.
```

Comment:
(just musing, think the product implications of doing this would be out of scope of interrupt)